Application/components/Messaging/NewTextChannelModal.tsx

```diff
@@ -42,6 +53,7 @@ export function NewTextChannelModal() {
   const [errorMessages, setErrorMessages] = useState({ channelName: '', members: '', admins: '' });
   const [senderID, setSenderID] = useState<string>('');
   const friends = useFriendList({lastFetched, setLastFetched});
+  const [captureHistory, setcaptureHistory] = useState(true); // On by default
```

Comment:
-  The default setting for captureHistory (message history capture) is set to true without explicit user input. This could lead to users unintentionally sharing sensitive information in the chat history. Consider setting a more privacy-conscious default or prompting users to confirm their privacy preferences explicitly.